.github/workflows/go.yml

```diff
@@ -87,642 +90,81 @@ jobs:
-          echo "cachePrefix=k0s-${PR_NUMBER}-${{ github.sha }}" >> $GITHUB_ENV
-      - name: Check out code into the Go module directory
-        uses: actions/checkout@v2
-
-      - name: Set up Go 1.x
-        uses: actions/setup-go@v2
-        with:
-          go-version: ${{ env.GO_VERSION }}
-        id: go
-
-      - name: Cache compiled binary for smoke testing
-        uses: actions/cache@v2
-        id: restore-compiled-binary
-        with:
-          path: |
-            k0s
-          key: build-${{env.cachePrefix}}
-
-      - name: Run smoke test .
-        run: make -C inttest check-etcd
-
-      - name: Collect test logs
-        if: failure()
-        uses: actions/upload-artifact@v2
-        with:
-          path: |
-            /tmp/*.log
-
-  smoketest-dualstack:
-    name: Smoke test for IPv6 dualstack
-    needs: build
-    runs-on: ubuntu-latest
-
-    steps:
-      - name: Get PR Reference and Set Cache Name
-        run: |
-          PR_NUMBER=$(echo ${GITHUB_REF} | cut -d / -f 3 )
-          echo "cachePrefix=k0s-${PR_NUMBER}-${{ github.sha }}" >> $GITHUB_ENV
-      - name: Check out code into the Go module directory
-        uses: actions/checkout@v2
-
-      - name: Set up Go 1.x
-        uses: actions/setup-go@v2
-        with:
-          go-version: ${{ env.GO_VERSION }}
-        id: go
-
-      - name: Cache compiled binary for smoke testing
-        uses: actions/cache@v2
-        id: restore-compiled-binary
-        with:
-          path: |
-            k0s
-          key: build-${{env.cachePrefix}}
-
-      - name: Run smoke test .
-        run: make -C inttest check-dualstack
-
-      - name: Collect test logs
-        if: failure()
-        uses: actions/upload-artifact@v2
-        with:
-          path: |
-            /tmp/*.log
-
-  smoketest-multicontroller:
-    name: Smoke test for multi controller
-    needs: build
-    runs-on: ubuntu-latest
-
-    steps:
-      - name: Get PR Reference and Set Cache Name
-        run: |
-          PR_NUMBER=$(echo ${GITHUB_REF} | cut -d / -f 3 )
-          echo "cachePrefix=k0s-${PR_NUMBER}-${{ github.sha }}" >> $GITHUB_ENV
-      - name: Check out code into the Go module directory
-        uses: actions/checkout@v2
-
-      - name: Set up Go 1.x
-        uses: actions/setup-go@v2
-        with:
-          go-version: ${{ env.GO_VERSION }}
-        id: go
-
-      - name: Cache compiled binary for smoke testing
-        uses: actions/cache@v2
-        id: restore-compiled-binary
-        with:
-          path: |
-            k0s
-          key: build-${{env.cachePrefix}}
-
-      - name: Run multicontroller test
-        run: make -C inttest check-multicontroller
-
-      - name: Collect test logs
-        if: failure()
-        uses: actions/upload-artifact@v2
-        with:
-          path: |
-            /tmp/*.log
-
-  smoketest-airgap:
-    name: Smoke test for airgap install
-    needs: build
+  build_windows:
```

Comment:
is it intention?

Reply:
trying to figure out where that change comes from?

git blame says a6b3153dcf185d92be99438e908efb33305e4943

Reply:
I think this is a result of how `git diff` calculates the diff.